bin/diagnostics.bash

```diff
@@ -41,82 +42,229 @@ sops_encrypt_file() {
 }
 
 run_diagnostics() {
+    # -- ck8s --
+    echo "Fetching CK8S software versions"
+    printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+    if [ -d "${CK8S_CONFIG_PATH}/capi" ]; then
+        # shellcheck disable=SC2002
+        capi_version=$(cat "${CK8S_CONFIG_PATH}"/capi/defaults/values.yaml | yq4 '.clusterApiVersion')
+        echo "CAPI version: ${capi_version}"
+    elif [ -d "${CK8S_CONFIG_PATH}/sc-config" ]; then
+        # shellcheck disable=SC2002
+        kubespray_version=$(cat "${CK8S_CONFIG_PATH}"/sc-config/group_vars/all/ck8s-kubespray-general.yaml | yq4 '.ck8sKubesprayVersion')
```

Comment:
Does it also work in multi-wc environments? And would it also work if they change their naming?

Reply:
I can't say it does for sure, will need to be tested.

Reply:
It should work for multi wc since that works by changing config path so each config path should always have a sc-config folder (one symlinked to the other). Different naming wont work but I don't think we support that in other places either.

Reply:
Right! Thanks! :pray: